packages/x-components/src/x-modules/facets/__tests__/utils.spec.ts

```diff
@@ -1,68 +1,4 @@
-import { EditableNumberRangeFilter, SimpleFilter } from '@empathyco/x-types';
-import { areFiltersDifferent, isNewQuery } from '../utils';
```

Comment:
`areFiltersDifferent`  was moved to `packages/x-components/src/utils/filters.ts`